www_src/pages/project/project.jsx

```diff
@@ -43,39 +43,74 @@ var Map = React.createClass({
       });
     });
   },
+
   componentDidMount: function () {
     var el = this.getDOMNode();
     var bounding = this.refs.bounding;
     var boundingEl = bounding.getDOMNode();
```

Comment:
looking at what we do with this var, as well as `scaleEl`, rewriting this to using setState+render to take care of the correct styling as you're interacting (see the `Positionable.jsx` component) means we don't need to even extract these two DOM nodes.